test/test_core.py

```diff
@@ -5546,8 +5546,8 @@ def test_fcntl(self):
   @also_with_nodefs_both
   @crossplatform
   def test_fcntl_open(self):
-    if '-DNODERAWFS' in self.emcc_args and WINDOWS:
-      self.skipTest('currently failing under windows and noderawfs')
+    if '-DNODERAWFS' in self.emcc_args and not LINUX:
+      self.skipTest('noderawfs fails here under non-linux')
```

Comment:
Why didn't this fail in emscripten CI?   We run all the crossplatform tests on macOS there I believe?